cefpanda/__init__.py

```diff
@@ -289,6 +313,10 @@ def _handle_mouse(self, mouseup):
             return
 
         posx, posy = self._get_mouse_pos()
```

Comment:
Does `posx` and `posy` need to be transformed by the scale/size?

Reply:
Yes, and position. `mouse_pos.html` demonstrates it.

Reply:
Er... Yes?